frontend/src/components/cluster/KubeConfigLoader.tsx

```diff
@@ -176,12 +176,15 @@ function KubeConfigLoader() {
           ...new Uint8Array(reader.result as ArrayBuffer),
         ]);
         const doc = yaml.load(data) as kubeconfig;
-        if (!doc.clusters || !doc.contexts) {
-          throw new Error('Invalid kubeconfig file');
+        if (!doc.clusters) {
+          throw new Error('Invalid kubeconfig file: No clusters found');
```

Comment:
We can make these strings translatable, is what I meant. `t('tanslation|No clusters found!')`. Then we can use this in the setError. Something like:
```ts
setError(t('translation|Invalid kubeconfig file: {{ errorMessage }}', {errorMessage: error.message}))
```

Reply:
Makes sense, pushed changes and included a screenshot of the new error message in the PR description